omod/src/main/java/org/openmrs/module/isantepluspatientdashboard/fragment/controller/IsantePlusMostRecentVitalsFragmentController.java

```diff
@@ -97,7 +97,7 @@ private String getBloodPressure(Patient patient) {
 				.getLatestSystolicBloodPressureForPatient(patient);
 
 		if (dBP != null && sBP != null) {
-			bp = String.valueOf(dBP.getValueNumeric()) + "/" + String.valueOf(sBP.getValueNumeric());
+			bp = String.valueOf(sBP.getValueNumeric()) + "/" + String.valueOf(dBP.getValueNumeric());
```

Comment:
i already made this change and pushed it to github, exclude this to avoid conflicts while merging